src/views/AnimalList.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from 'react';
+import { fetchAnimals } from '../services/fetch';
+import DropDown from '../components/DropDown';
+
+export default function AnimalList() {
+  const [animals, setAnimals] = useState([]);
+  const [type, setType] = useState([]);
+  // const [selectedType, setSelectedType] = useState([]);
+
+  const [loading, setLoading] = useState(true);
+
+  //
+
+  useEffect(() => {
+    const getAnimals = async () => {
+      try {
+        const data = await fetchAnimals();
+
+        setAnimals(data);
+        console.log(data);
+
+        setTimeout(() => {
+          setLoading(false);
+        }, 750);
+      } catch (e) {
+        setErrorMessage(
+          'Woops...something went wrong. Please refresh the page.'
+        );
+        setLoading(false);
+      }
+    };
+    getAnimals();
+  }, [type]);
+
+  // useEffect(() => {
+  //   setType(animals);
+  // }, [selectedType]);
+
+  const filterAnimals = (animal) => {
+    setType(
+      animals.filter((animal) => animal.animal_type === type || type === 'All')
+    );
+  };
```

Comment:
It appears our drop down isn't filtering by animal_type, as reptiles and mammals appear when filtering by mammals. I believe this is due to setting type and checking if animal_type is equal to type. This overlaps our code and causes some strange behavior in the filter.